.github/teams.yml

```diff
@@ -9,3 +9,112 @@ test:
 area/alerting:
   channel-label: C028MCV4R7C
   exclude-github-team: alerting-squad
+
+# DataViz squad
+area/dataviz:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/legend:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/barchart:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/bargauge:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/candlestick:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/canvas:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/common:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/common:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/gauge:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/geomap:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/graph:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/heatmap:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/histogram:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/icon:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/icon:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/piechart:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/stat:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/state-timeline:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/status-history:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/timeseries:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/trend:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/xychart:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/panel/xychart:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/tooltip:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
+
+area/transformations:
+  channel-label: C04J73AAQ87 # grafana-dataviz
+  exclude-github-team: dataviz-squad
```

Comment:
this one is probably BI, but still relevant. i'm assuming we cannot do both? though i can assign both projects in the UI :thinking:

Reply:
I'm not sure if we can assign multiple teams to a label 🤔 I think we can experiment with this approach and if it works well then talk to BI about implementing a similar approach for things they own (as well as other squads)